RxSwift/Observables/Observable+Single.swift

```diff
@@ -84,7 +84,7 @@ extension ObservableType {
     - returns: The source sequence with the side-effecting behavior applied.
     */
     // @warn_unused_result(message:"http://git.io/rxs.uo")
-    public func doOn(eventHandler: (Event<E>) throws -> Void)
+    public func doOn(_ eventHandler: (Event<E>) throws -> Void)
```

Comment:
Hmm, should I open issue about that and let's see how it goes? Seems like many people wanted the shortcut syntax 😄